examples/example4.py

```diff
@@ -0,0 +1,38 @@
+"""Watch multiple K8s event streams without threads."""
+import asyncio
+from kubernetes_asyncio import client, config, watch
```

Comment:
Travis is configured to run tests and check style of code. I know it's crazy... but he complains about a missing blank line here - between standard imports and local/library imports. Can you add this ?

Logs https://travis-ci.org/tomplus/kubernetes_asyncio/jobs/389560197 
Status is presented on PR page and it's checked on every commit.

Reply:
I copied all scripts from the official python client (https://github.com/kubernetes-client/python) to make it as similar as possible to it.